docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `authorityKeyIdentifier`          | MUST         | N            | See [Section 7.1.2.9.2](#71292-authority-key-identifier) |
+| `basicConstraints`                | MUST         | Y            | See [Section 7.1.2.6.8](#71268-basic-constraints) |
+| `certificatePolicies`             | MUST         | N            | See [Section 7.1.2.6.9](#71269-certificate-policies) |
+| `extKeyUsage`                     | MUST         | -            | See [Section 7.1.2.6.10](#712610-extended-key-usage) |
+| `subjectAltName`                  | MUST         | -            | See [Section 7.1.2.6.12](#712612-subject-alternative-name) |
+| `nameConstraints`                 | MUST NOT     | -            | - |
+| `crlDistributionPoints`           | MAY          | N            | See [Section 7.1.2.9.3](#71293-crl-distribution-points) |
+| `keyUsage`                        | MAY          | Y            | See [Section 7.1.2.6.11](#712611-key-usage) |
+| CA/Browser Forum Onion Extension  | MAY          | N            | __**TODO**__ |
+| Signed Certificate Timestamp List | MAY          | N            | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
+| `subjectKeyIdentifier`            | SHOULD NOT   | N            | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| Any other extension               | SHOULD NOT   | -            | __**TBD**__ |
+
+##### 7.1.2.6.7 Authority Information Access
+
+The `AuthorityInformationAccesssSyntax` MUST contain one or more `AccessDescription`s. Each `AccessDescription` MUST only contain a permitted `accessMethod`, as detailed below, and each `AuthorityInformationAccessSyntax` MUST contain all required `AccessDescription`s.
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MUST         | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's OCSP responder. |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD       | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's certificate. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.6.8 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be FALSE |
+| `pathLenConstraint` | MUST NOT be present |
+
+**Note**: CAs MUST observe DER encoding rules, such as not explicitly encoding DEFAULT values within OPTIONAL fields.
+
+##### 7.1.2.6.9 Certificate Policies
+
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
```

Comment:
Why does it have to be "The first"?  The BRs don't currently mandate any specific order.
(Sectigo's current implementation adds the Reserved Certificate Policy Identifier last, rather than first).

Reply:
Good question!

This was an attempt to provide clear and unambiguous guidance regarding which policies MUST be present. I can totally explore a different approach here, although I do want to call out that the ordering totally has impact on applications and how they verify. For example, for Firefox and Chrome, the ordering of the OID [totally matters](https://wiki.mozilla.org/CA/EV_Processing_for_CAs#First_OID), and it can also impact how macOS determines EV status.

Because of this, I think it's useful to provide clear and consistent guidance to help avoid these issues if folks are concerned with this change. I tried to balance "uncontroversial clarity improvements" (which would require only a short phase-in) with those which may have more complicated interactions.

Reply:
@benwilson-mozilla pointed out https://wiki.mozilla.org/CA/EV_Processing_for_CAs#First_OID in https://github.com/mozilla/pkipolicy/issues/248

@robstradling What does Sectigo do for EV policies?

Reply:
@sleevi Currently we always include the Sectigo EV OID first (1.3.6.1.4.1.6449.1.2.1.5.1) in the Certificate Policies extension, and the CABForum EV OID second (2.23.140.1.1).  Example: https://crt.sh/?id=4994988183

So far we've only ever asked the various EV root programs to associate the Sectigo EV OID with our roots, although I'm aware that some (but not all) browsers implicitly associate 2.23.140.1.1 with all EV-enabled roots.

Reply:
@robstradling Thanks!

The (but not all) is Chrome and Safari, right? That is, IIRC, Mozilla and Microsoft grant the CABF OID for all EV.

Or is it just Chrome at this point?

Reply:
It's just Chrome at this point (or soon will be if not).

Reply:
@sleevi authroot.stl explicitly grants 2.23.140.1.1 to a bunch of EV roots, presumably because those CAs have asked Microsoft to do that.  This would seem to imply that Microsoft doesn't implicitly grant 2.23.140.1.1 to all EV roots, because otherwise those explicit grants would be pointless.

I'm aware that Apple (https://github.com/apple-oss-distributions/security_certificates/blob/main/CertificateTool/BuildOSXAsset/buildRootKeychain.rb#L593) and Mozilla (https://bugzilla.mozilla.org/show_bug.cgi?id=1243923#c0) both implicitly grant 2.23.140.1.1 to all EV roots, although I would have to dig further to figure out the earliest MacOS and Firefox versions that have this behaviour (and after that to estimate the % of users that are still using even earlier versions).

Reply:
BTW, re: https://wiki.mozilla.org/CA/EV_Processing_for_CAs#First_OID, see https://bugzilla.mozilla.org/show_bug.cgi?id=1769150

Reply:
For Apple, it's a very recent change that the CA/BF OID was granted to all EV roots. I can pin down the exact version (I think iOS 15.5, macOS 12.4) if it's helpful, but it's within the last ~6 weeks.

Reply:
@clintwilson That surprises me.  https://github.com/apple-oss-distributions/security_certificates/commit/62da197038ec7adfa79496cddf33231b0fe36b31#diff-a6d1c50aa472a84fa67cbc2a839339862f07f49db49289d85233caf2941bda9eR593 was added in security_certificates-55161.0.3, which corresponds to macOS 10.15.  Have I misunderstood the purpose of that line of Ruby code?

Reply:
> @sleevi authroot.stl explicitly grants 2.23.140.1.1 to a bunch of EV roots, presumably because those CAs have asked Microsoft to do that. This would seem to imply that Microsoft doesn't implicitly grant 2.23.140.1.1 to all EV roots, because otherwise those explicit grants would be pointless.

I thought it was a recent-ish change (past 2y) that added explicit grants. That is, it's not implicit in code, but implicit that if you're granted EV treatment, it includes the CABF OID. But I haven't double-checked to confirm, so I very well could be misremembering.

But yeah, this discussion has been helpful, because it's highlighted:
1. If the CABF OID _isn't_ granted, then applications that use the first-EV-oid-logic could get the wrong result if the root wasn't enabled for CABF. This suggests that it may be good for Chrome to change and grant CABF to the EV roots, in order to facilitate an (eventual) transition to CABF-only OIDs
2. Even if the CABF OID _is_ granted, there may be legacy reasons, **for EV certificates only**, to use an EV first
3. A transition date is necessary here, regardless

The "goal" trying to be captured here is to ensure alignment around CABF OIDs and consistent, unambiguous processing of those. However, I'll rework this, such that:

- Either CABF || CA Reserved are allowed
- CABF is MUST be present, and SHOULD be first

For v1. A MUST be first may depend on external factors of browsers (such as browser updates and update rates), so it's not reasonable to push to CAs yet.

Reply:
@robstradling Thanks for raising this.

I took a stab at addressing this in https://github.com/sleevi/cabforum-docs/pull/36/commits/197f0964d991059609aa1acd7cb5eb6a53bfc343 which the commit description tries to explain what changed.